Dockerfile

```diff
@@ -14,9 +14,7 @@ LABEL maintainer="shenxianpeng <20297606+shenxianpeng@users.noreply.github.com>"
 
 RUN apt-get update && apt-get -y install python3-pip
 
-COPY cpp_linter/ pkg/cpp_linter/
-COPY pyproject.toml pkg/pyproject.toml
-RUN python3 -m pip install pkg/
+RUN python3 -m pip install cpp-linter
```

Comment:
We can add our version requirements here (if we ever need to).
```suggestion
RUN python3 -m pip install 'cpp-linter<1.5'
```

Reply:
We should have a version limit here

Reply:
I think it would only be needed if we introduce breaking changes. The version number would be specific to the cpp-linter pkg (not the docker/composite action version).

Reply:
Sure.